### ISSUE TITLE
Unable to linking during compilation.

### ISSUE THREAD

**Issue by Bouh** [ninja-build/ninja#1757]
Hello

We use ninja since few weeks on GDevelop 5, i've a issue with ninja for compile GD5. [See the original issue](https://github.com/4ian/GDevelop/issues/1591).
During the compilation process i'm stuck on this: 

![image](https://user-images.githubusercontent.com/1670670/77824283-d3c48880-7101-11ea-9295-897845efd066.png)

```
[145/294] Linking CXX static library ..\Output\Release_Emscripten\libGDCore.bc
FAILED: ../Output/Release_Emscripten/libGDCore.bc
cmd.exe /C "cd . && C:\emsdk\upstream\emscripten\emar.bat rc ..\Output\Release_Emscripten\libGDCore.bc  @CMakeFiles/GDCore.rsp && cd ."
C:/emsdk/upstream/bin\llvm-ar.exe: error: CoreCMakeFilesGDCore.dirGDCoreEventsBuiltinCommentEvent.cpp.o: no such file or directory
```

Like you can see the path is wrong for this file, all separators are missing.
The issue is always at this lines.
Also `/Output/Release_Emscripten/libGDCore.bc` isn't write, but other works.

Here the [complete log file](https://pastebin.com/raw/vyqMPx8W), you will be right path for other files.

If this file can help you:
GDevelop\Binaries\embuild\CMakeFiles\ [GDCore.rsp]

I've tried to use cmd and powershell as admin, same issue.
Iv'e made a clean install of everything, same result.
I use Cmake 3.5.0.

I need to be sure, if this thing is a ninja issue or not.
Thank for your help.

**Comment by jhasse** [comment]
> I use Cmake 3.5.0.

Can you try with CMake 3.17 and see if the issue persists?

**Comment by Bouh** [comment]
Issue is fixed with migration to CMake 3.17.
Thanks a lot!
